src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -758,6 +759,20 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		if (Skript.classExists("org.bukkit.event.entity.PiglinBarterEvent")) {
+			Skript.registerEvent("Piglin Barter", SimpleEvent.class, PiglinBarterEvent.class, "piglin (barter[ing]|trad(e|ing))")
+				.requiredPlugins("Minecraft 1.16.5 or newer")
+				.description(
+					"Called when a piglin finishes bartering. This event may be called when a piglin picks up an item on its bartering list.",
+					"Cancelling will prevent piglins from dropping items, but will still make them pick up the input.")
+				.examples(
+					"on piglin barter:",
+					"\tif piglin barter outcome contains diamond:",
+					"\t\tsend \"Diamonds belong in the money pit!\" to player",
+					"\t\tcancel event"
+				)
+				.since("INSERT VERSION");
+
```

Comment:
missing closing }

Reply:
To be fair this was caused by me resolving the conflicts in GitHub's Fantastic Million-Dollar web UI

Reply:
well done kenzie